lib/pg.cursor.js

```diff
@@ -33,8 +33,7 @@ class PostgresCursor extends Cursor {
 
   fetch(callback) {
     if (!this.category) {
-      callback(new Error('Category name was not specified'));
-      return;
+      throw new Error('Category name was not specified');
```

Comment:
Should we maybe throw `TypeError` here?
```suggestion
      throw new TypeError('Category name was not specified');
```